app/models/ability.rb

```diff
@@ -0,0 +1,14 @@
+class Ability
+  include CanCan::Ability
+
+  def initialize(user)
+    user ||= User.new
+
+    if user.admin?
+      can :destroy, Post
+      can :destroy, Comment
+    else
+      can :read, :all
+    end
```

Comment:
[REQUIRED] Kindly adjust your code. Users are supposed to be able to delete posts and comments if it belongs to them.

Reply:
Thanks, @fobadara . Almost forgot to include that condition!